exampleSite/content/docs/guide/shortcodes/icon.fa.md

```
@@ -20,28 +20,35 @@ enableInlineShortcodes: true
 {{< icon "gift" >}}
 {{< icon "sparkles" >}}
 
-## استفاده
+## نحوه استفاده
 
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)